### ISSUE TITLE
Auto inserting bullets failed

### ISSUE THREAD

**Issue by digizeph** [plasticboy/vim-markdown#224]
For editing bullets, when I finished one bullet, hit enter, it will give me a 4-space tab instead of a new bullet as I hoped. I am not sure if that's a coding issue or my configuration issue.

I enabled the following two options in vimrc, nothing else.
```
let g:vim_markdown_folding_disabled=1
let g:vim_markdown_frontmatter=1
```

**Comment by cirosantilli** [comment]
Yes, this is a long standing "what is the best default" problem that needs to be solved. See also: https://github.com/plasticboy/vim-markdown/issues/139

**Comment by simplyianm** [comment]
have the same problem. How do I fix this?
